_IBC/uptick-cosmos.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "uptick",
```

Comment:
for both ibc files, they must be ordered alphabetically, so they should be named cosmoshub-uptick.json, with chain 1 being cosmoshub, and chain 2 being uptick